File: src/test/java/cc/sukazyo/cono/morny/MornyCLI.java
package cc.sukazyo.cono.morny;

import cc.sukazyo.cono.morny.util.UniversalCommand;

import java.util.*;

public class MornyCLI {
	
	public static void main (String[] args) {
		
		System.out.print("$ java -jar morny-coeur-"+MornySystem.VERSION_FULL()+".jar " );
		String x;
		try (Scanner line = new Scanner(System.in)) { x = line.nextLine(); }
		ServerMain.main(UniversalCommand.format(x));
		
	}
	
}

File: src/test/scala/cc/sukazyo/cono/morny/MornyCLI.scala
package cc.sukazyo.cono.morny

import cc.sukazyo.cono.morny.util.CommonFormat

import java.time.Duration
import java.time.temporal.ChronoUnit

@main def MornyCLI (): Unit = {
	
	val duration = Duration.ZERO
//		.plusDays(2)
//		.plusHours(21)
//		.plusMinutes(2)
//		.plusSeconds(5)
		.plusMillis(123)
		.plusNanos(876548)
	
	val echo = CommonFormat.formatDurationTimers(duration, ChronoUnit.SECONDS)
	
	println(echo)
	
}
